test/e2e/app-dir/dynamic-io-errors/fixtures/lazy-module-init/app/server/page.tsx

```diff
@@ -4,13 +4,14 @@ export default async function Page() {
   return (
     <>
       <p>
-        This page has several client modules that have sync IO in the module
-        scope that will run in serial when prerendering to HTML. The point of
-        this test is to assert that the serial nature does not lead to
-        unexpected sync IO errors. In the past this particular setup would show
-        up as a sync IO error because the later modules did not initialize
-        during the prospective render and thus appeared to use sync IO when
-        being initialized during the final render.
```

Comment:
I think you intended to use this paragraph in `test/e2e/app-dir/dynamic-io-errors/fixtures/lazy-module-init/app/serial-client-sync-io/page.tsx`?

Reply:
yeah, I did :)